src/identity/email-password-identity-provider/default-views/repost.ejs

```diff
@@ -0,0 +1,45 @@
+<html>
+  <head>
+    <title>Fragment Federated Callback Repost Form</title>
+  </head>
+  <body>
+    <script type="text/javascript">
```

Comment:
Let's not have such inline code; it should be part of the JavaScript (or even TypeScript) codebase where it can go through all of the checks.

Why is the repost needed though?

Reply:
Again, left over from something before. I'll clean it out when I go through to make everything nice.